### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by QuentinRoy** [Zulko/eagle.js#22]
`eagle.js 0.1.2` does not include its dist folder on `npm`.

This is quite a problem as the current `main` field points toward a non-`require`-able un-transpiled module (c.f. #14).

@ktsn has done a great work in #15 in solving this issue. I am looking forward to use it.

**Comment by Zulko** [comment]
Sorry for being a bad maintainer :disappointed:  I'll push the new version asap, possibly this evening.

**Comment by QuentinRoy** [comment]
We all do what we can. :)
I really like the idea of using vue for slides. I agree with you it is perfect for it. Your system seems to combine readability and flexibility in a way I have not find with the other I have tried. I also really like the minimalistic/hackable approach. I often found myself very constrained by `Reveal.js` for example.
I wish I had the time to contribute (I realized eagle.js does not have a presenter view, and unfortunately, that is a deal breaker for me 😓).
